samples/charts/category-chart/annotations/wwwroot/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
you must have some VS Code extension that keeps inserting not needed whitespace in .html files